semgrep/Makefile

```diff
@@ -7,3 +7,11 @@ test:
 install:
 	cp -rv ./build/semgrep.dist/ /usr/local/bin/semgrep-files/
 	ln -sf /usr/local/bin/semgrep-files/semgrep /usr/local/bin/semgrep
+
+.PHONY: qa-test
+qa-test: build
+	pytest -s tests/acceptance/qa.py
+
+.PHONY: regenerate-tests
+regenerate-tests:
```

Comment:
Does this do anything right now? I assume we'd have to have a `__main__` check in `qa.py` for this do things.

Reply:
You're right this is stale. Removing.